ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -40,6 +40,7 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.10
+test_binary_build_commands: chmod uga+w /etc/passwd
```

Comment:
ok, this was wrong, my understanding is that `test_binary_build_commands` is ran for the `test-bin` image but in `e2e-aws-go` we're not using `test-bin` (there's no `container: from: test-bin`. I don't believe we can use `binary_build_commands` as we don't seem to be using the `bin` image either afaict (just plan `src`?) (for reference https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#binary_build_commands)

not sure what to do actually, @stevekuznetsov @smarterclayton wdyt?

Reply:
ok we're using `openshift_installer_src` so we're using the `src` container by default anyway for `e2e-aws-go`, that's why `chmod` is not executed nor it would be using `binary_build_commands`.

Reply:
Meanwhile, https://github.com/openshift/release/pull/2819